### PR TITLE
(chore) Fix husky by using postinstall hook

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
According to the statement on the Husky website [here](https://typicode.github.io/husky/#/?id=install), it states that
```
Yarn 2+ doesn't support prepare lifecycle script, so husky needs to be installed differently (this doesn't apply to Yarn 1 though). See [Yarn 2+ install](https://typicode.github.io/husky/#/?id=yarn-2).
```
Hence I have incorporated the changes according to the instructions [here](https://typicode.github.io/husky/#/?id=yarn-2).

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-appointments-app/src/appointment-forms/appointments-form.component.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ interface AppointmentFormProps {
   patientUuid?: string;
   context: string;
 }
-const AppointmentForm: React.FC<AppointmentFormProps> = ({ appointment = {}, patientUuid, context }) => {
+const AppointmentForm: React.FC<AppointmentFormProps> = ({ appointment, patientUuid, context }) => {
```
Comment (ibacher) [inline packages/esm-appointments-app/src/appointment-forms/appointments-form.component.tsx:56]: This change isn't going to break anything is it?

**Review by ibacher — APPROVED** [review]
Nice catch @vasharma05!

File: packages/esm-appointments-app/src/appointment-forms/appointments-form.component.tsx (hunk 1 above)
Comment (hadijahkyampeire) [reply packages/esm-appointments-app/src/appointment-forms/appointments-form.component.tsx:56]: It is not I confirmed it.

File: package.json [hunk 2]
```diff
@@ -58,6 +58,7 @@
     "jest-environment-jsdom": "^28.1.0",
     "lerna": "^3.20.2",
     "openmrs": "latest",
+    "pinst": "^3.0.0",
```
Comment (ibacher) [inline package.json:61]: Do we need this dependency? My guess is no for this setup (since the monorepo package is not published.

**Review by hadijahkyampeire — APPROVED** [review]
Thanks @vasharma05

File: package.json (hunk 2 above)
Comment (vasharma05) [reply package.json]: Thank you @ibacher for the catch, I removed it.

**Pull request merged**